sagemaker-jumpstart/nlp_score_dashboard_sec/Dashboard_SEC_Filings.ipynb

```diff
@@ -0,0 +1,1094 @@
+{
```

Comment:
<p>Are you able to add the names of the third-party packages this is referring to?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3016/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>

Reply:
<p>I think it is listed below with </p><p> </p><pre class="ql-syntax" spellcheck="false">R v3.5.2: GPLv3 license (https://www.gnu.org/licenses/gpl-3.0.html)
DT v0.19.1: GPLv3 license (https://github.com/rstudio/DT/blob/master/LICENSE)
</pre>